homeassistant/components/samsungtv/media_player.py

```diff
@@ -111,17 +113,11 @@ class SamsungTVDevice(MediaPlayerDevice):
 
     def __init__(self, host, port, name, timeout, mac, uuid):
         """Initialize the Samsung device."""
-        from samsungctl import exceptions
-        from samsungctl import Remote
-        import wakeonlan
 
         # Save a reference to the imported classes
-        self._exceptions_class = exceptions
```

Comment:
Don't delete this lines
```
self._exceptions_class = samsung_exceptions
self._remote_class = SamsungRemote
self._wol = wakeonlan
```

Reply:
It may have been very fast in my review ...
I'm not sure now

Reply:
Hi @javicalle !

Is it usual in Python to store an external class like this and reuse it later on ?

I think it shouldn't so that's why, but I will appreciate help on tests.

Thanks

Reply:
Hi @Quentame,
 I don't know because I'm not fluent in python, but for sure is not usual at OOP.

I am reviewing it and I think there is a problem in the remote Mock (there are a few of them):
https://github.com/home-assistant/home-assistant/blob/5fd9b474dca3c9eaca6ad6f86625cc54ba88175f/tests/components/samsungtv/test_media_player.py#L125-L135

Unfortunately I don't know how to fix the error.

Reply:
Change "samsungctl.Remote" to "homeassistant.components.samsungtv.media_player.SamsungRemote" in the patch.

Reply:
Hi @MartinHjelmare !

I can finally run my tests locally, will be easier for next PRs.

So no more "Fix test [n+1]" 😄 

Enjoy !